nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -51,6 +51,17 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
   `CONFIG_FILE_NAME` includes `bpf_pinning`, `ematch_map`, `group`, `nl_protos`, `rt_dsfield`, `rt_protos`, `rt_realms`, `rt_scopes`, and `rt_tables`.
 
+- switch-to-configuration does not directly call systemd-tmpfiles anymore.
+  Instead the new artificial sysinit-reactivation.target is introduced which
+  allows to restart multiple services that are ordered before sysinit.target
+  and respect the ordering between the services.
+  `systemd-tmpfiles-setup.service` now sets `RefuseManualStop = false` so that
+  the unit can be restarted by switch-to-configuration. When upgrading via a
+  system switch, however, this will lead to a failure because on the previous
+  generation it is still `RefuseManualStop = true`. See [this
+  issue](https://github.com/NixOS/nixpkgs/pull/271067#issuecomment-1845714645)
+  for context. A reboot will solve this issue.
```

Comment:
This bit about `RefuseManualStop` is not actually true anymore in the current state of this PR.